tutorials/transfer_learning/training.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import print_function
```

Comment:
This file doesnt even import tvm and is pure pytorch. why are we pulling in pytorch code?

Reply:
I see.